index.ts

```diff
@@ -452,7 +455,7 @@ export const configuration = configure(async sdm => {
             ],
         },
         shadowCljs: {
-            test: [not(repoSlugMatches(/^atomisthq\/admin-app$/)), ShadowCljsPushTest],
+            test: [not(or(repoSlugMatches(/^atomisthq\/admin-app$/), repoSlugMatches(/^atomist-skills/))), ShadowCljsPushTest],
```

Comment:
No need to call it twice or import `or`.

```suggestion
            test: [not(repoSlugMatches(/^(?:atomist-skills/.*|atomisthq\/admin-app)$/)), ShadowCljsPushTest],
```